resources/views/internlink/policy.blade.php

```diff
@@ -0,0 +1,21 @@
+@extends('layouts.panel')
+@section('panel-cols', '10')
+@section('panel-offset', '1')
+@section('panel-heading', 'InternLink')
+@section('panel-body')
+    <h2>Usage Policy</h2>
+    <h4>InternLink is meant to be a tool for current CUMSA members (Users) to seek the advice of past/current members (Contributors)
+        with regards to internship and job applications. Below are the privacy policy for the platform.</h4>
+    <h3><u>For contributors</u></h3>
+    <h4>CUMSA will store the information you have provided on a server and make it available only to members of CUMSA (by authentication via RAVEN).
```

Comment:
* (by authentication via RAVEN) -> could this be removed? Especially since Raven will not be the only authentication method - username/password will be used for alumni...